packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -603,15 +595,7 @@ describe('snippet-bot', () => {
         .get(
           '/repos/tmatsuo/repo-automation-bots/issues/14/comments?per_page=50'
         )
-        .reply(200, [])
-        .post(
-          '/repos/tmatsuo/repo-automation-bots/issues/14/comments',
-          body => {
-            snapshot(body);
-            return true;
-          }
-        )
-        .reply(200);
+        .reply(200, []);
```

Comment:
Would it be possible to add a test for the two new cases:

"no comment added if no samples modified"

"existing comment updated, if comment existed and samples no longer updated".

Reply:
Added 2 tests, PTAL